examples/wearable/UIComponents/contents/navigationelements/index.html

```diff
@@ -41,8 +41,8 @@ <h2 class="ui-title">
 					</a>
 				</li>
 				<li>
-					<a href="drawer.html">
-						Drawer
+					<a href="assist-panel.html">
+						Assist Panel
```

Comment:
to consider: in guides it says Primary & Assist Panel

Reply:
What means "Primary"? Is this means Primary Panel and Assist Panel. We can change this when we will have more guidelines.

Reply:
ok